### PR TITLE
chore(docs): add bep support

### PR DESCRIPTION
**Pull request by alxhotel**
**What is the purpose of this pull request? (put an "X" next to item)**

[X] Documentation update
[ ] Bug fix
[ ] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
Add a list of the BEPs supported by Webtorrent.

**Which issue (if any) does this pull request address?**
None.

**Is there anything you'd like reviewers to focus on?**
Check that the list is correct.

### CODE REVIEW

File: docs/bep_support.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+| UDP Tracker Protocol | [BEP 15](https://www.bittorrent.org/beps/bep_0015.html) | :heavy_check_mark: | :x: |
+| WebSeed - HTTP/FTP Seeding (GetRight style) | [BEP 19](https://www.bittorrent.org/beps/bep_0019.html) | :heavy_check_mark: | :heavy_check_mark: |
+| Tracker Returns Compact Peer Lists | [BEP 23](https://www.bittorrent.org/beps/bep_0023.html) | :heavy_check_mark: | :heavy_minus_sign: |
+| Private Torrents | [BEP 27](https://www.bittorrent.org/beps/bep_0027.html) | :heavy_check_mark: | :heavy_check_mark: |
+| uTorrent transport protocol (uTP) | [BEP 29](https://www.bittorrent.org/beps/bep_0029.html) | :heavy_check_mark: | :x: |
+| DHT Extensions for IPv6 | [BEP 32](https://www.bittorrent.org/beps/bep_0032.html) | :heavy_minus_sign: [bittorrent-dht#88](https://github.com/webtorrent/bittorrent-dht/issues/88) | :heavy_minus_sign: |
+| Updating Torrents Via DHT Mutable Items | [BEP46](https://www.bittorrent.org/beps/bep_0046.html) | :heavy_minus_sign: [webtorrent#886](https://github.com/webtorrent/webtorrent/issues/886) | :heavy_minus_sign: [webtorrent#886](https://github.com/webtorrent/webtorrent/issues/886) |
```
Comment (feross) [inline docs/bep_support.md:25]: Small formatting issue: add a space between BEP and 46.

File: docs/bep_support.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+:x: = cannot be implemented
+
+| Name | Link | Node.js | Browser |
+|---|---|---|---|
+| Distributed hash table (DHT) | [BEP 5](https://www.bittorrent.org/beps/bep_0005.html) | :heavy_check_mark: | :heavy_minus_sign: |
+| IPv6 Tracker Extension | [BEP 7](https://www.bittorrent.org/beps/bep_0007.html) | :heavy_minus_sign: | :heavy_minus_sign: |
+| Magnet links | [BEP 9](https://www.bittorrent.org/beps/bep_0009.html) | :heavy_check_mark: | :heavy_check_mark: |
+| Extension Protocol | [BEP 10](https://www.bittorrent.org/beps/bep_0010.html) | :heavy_check_mark: | :heavy_check_mark: |
+| Peer Exchange (PEX) | [BEP 11](https://www.bittorrent.org/beps/bep_0011.html) | :heavy_minus_sign: [webtorrent#1191](https://github.com/webtorrent/webtorrent/issues/1191) | :heavy_minus_sign: [webtorrent#1191](https://github.com/webtorrent/webtorrent/issues/1191) |
```
Comment (feross) [inline docs/bep_support.md:17]: We do support PEX in Node.js for TCP peers.

File: docs/bep_support.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# BEP Support
+
+These are the [BitTorrent Extension Protocols (BEP)](https://www.bittorrent.org/beps/bep_0001.html) supported by Webtorrent:
+
+:heavy_check_mark: = implemented
```
Comment (feross) [inline docs/bep_support.md:5]: I think  ✅  looks better and stands out more from :heavy_minus_sign:

**Review by feross — APPROVED** [review]
LGTM!

**Comment by alxhotel** [comment]
Thanks, @feross! I've updated it

**Review by DiegoRBaquero — APPROVED** [review]

**Pull request merged**